### ISSUE TITLE
Fix Heading Levels

### ISSUE THREAD

**Issue by cncoulter** [cncoulter/cncoulter.github.io#4]
Heading levels are a bit of a mess. Use only 1 h1 per page, use headings in a logical order (1 to 6), and don't skip levels.

1. Your blog post pages skip heading level 1.
2. Your homepage includes two h1 elements.
3. Your homepage skips to heading level 5 for the sidebar.

**Comment by cncoulter** [comment]
This one is tricky.

On the blog post pages, you want the post title to be h1, and all headings within the post to start at h2. However, on the home page, the "Hiya!" heading should be h1, "Blog" should be h2, the post title should then be h3, and all headings within the post should start at h4.

I don't know how to change the heading levels depending on which page it is (a blog post page or the home page). There might be a way to do that with JavaScript? But I'm more of an HTML/CSS person than a JavaScript person, so that's outside my skill set right now.

Here's the solution that I'm planning on:

* Update all blog posts so that headings within a post start at h2
* Update the blog (archive) page to display just post titles and descriptions, rather than the full post

I'm planning to execute this solution as part of a redesign of the site, where I shift away from using Bootstrap and instead write all the CSS myself. It'll happen, but it'll take some more time.

**Comment by cncoulter** [comment]
Partial fix!

https://github.com/cncoulter/cncoulter.github.io/commit/64137a0cb02176c654c4a70fba14b4ed1ed2d599

The complete fix is in progress at <https://www.camcoulter.com/>
